react/components/AccordionFilterItem.js

```diff
@@ -4,9 +4,8 @@ import { intlShape, injectIntl } from 'react-intl'
 import classNames from 'classnames'
 import { Tag } from 'vtex.styleguide'
 import { useCssHandles } from 'vtex.css-handles'
-import { IconCaret } from 'vtex.store-icons'
```

Comment:
Doesn't this break the app? 🤔 
This component is being used at line 89

Reply:
@victorhmp Sorry about that, I've replaced :s